src/__tests__/iterate-focusable-elements.test.tsx

```diff
@@ -57,3 +97,30 @@ it('Should iterate through focusable elements in reverse', () => {
   expect(focusable[3].tagName.toLowerCase()).toEqual('input')
   expect(focusable[4].tagName.toLowerCase()).toEqual('textarea')
 })
+
+it('Should ignore hidden elements if strict', async () => {
+  const {container} = render(
+    <div>
+      <button>Apple</button>
+      <button style={{visibility: 'hidden'}}>Banana</button>
+      <button style={{display: 'none'}}>Watermelon</button>
+      <div style={{visibility: 'hidden'}}>
+        <div>
+          <button>Cherry</button>
+        </div>
+      </div>
+      <div style={{display: 'none'}}>
+        <div>
+          <button>Peach</button>
+        </div>
+      </div>
+      <button tabIndex={-1}>Cantaloupe</button>
```

Comment:
Maybe I'm missing something. Doesn't this make it non focusable with keyboard? How does this button have a place in the `focusable` array?

Reply:
The `iterateFocusableElements` will return "[any element with `tabindex` explicitly set can be focusable, even if it's set to "-1"](https://github.com/primer/behaviors/blob/jellobagel-focus-zone-options/src/utils/iterate-focusable-elements.ts#L111)" if `onlyTabbable` option is false